lib/main_containers.pm

```diff
@@ -393,7 +393,7 @@ sub load_container_tests {
 
     ## Helm chart tests. Add your individual helm chart tests here.
     if (my $chart = get_var('HELM_CHART')) {
-        loadtest 'containers/k3s_ensure_essentials_installed';
+        loadtest 'containers/k3s_ensure_essentials_installed' if (helm_is_supported() && check_var('HOST_VERSION', '15-SP7'));
```

Comment:
What is the rationale behind `if (helm_is_supported() && check_var('HOST_VERSION', '15-SP7'));`?